.github/workflows/release-swift-toolchain-binary-sizes.yml

```diff
@@ -73,7 +74,7 @@ jobs:
         run: echo "GOOGLE_APPLICATION_CREDENTIALS=${{ github.workspace }}/.google_application_credentials" | Out-File -FilePath $env:GITHUB_ENV -Append
 
       - name: Install Swift toolchain
-        uses: compnerd/gha-setup-swift@main
+        uses: compnerd/gha-setup-swift@9955d596781e4dda7bc4ca61bd534be03660b698
```

Comment:
This can also be changed to `gha-setup-swift@main`?

Reply:
Good point :P changed back!